hazelcast/proxy/set.py

```diff
@@ -36,6 +44,7 @@ def add(self, item):
         return self._invoke(request, set_add_codec.decode_response)
 
     def add_all(self, items):
+        # type: (List[Any]) -> Future[bool]
```

Comment:
`List[Any]` vs `list[Any]`, I see that we use both in the PR, we should be consistent and use the one that is supported by the older Python versions

Reply:
List[Any] is used for older Python versions, I corrected them.